.github/workflows/build-github-package.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         run: mvn -B package --file up-core-api/pom.xml
```

Comment:
I suppose this also needs to be updated? In fact, using the pom.xml file from the current working directory should not require the `--file` param at all.